serializer/inc/methodreturn.h

```diff
@@ -8,13 +8,10 @@ typedef struct METHODRETURN_HANDLE_DATA_TAG* METHODRETURN_HANDLE;
 
 #include "azure_macro_utils/macro_utils.h"
 
-/*the following macro expands to "const" if X is defined. If X is not defined, then it expands to nothing*/
-#define CONST_BY_COMPILATION_UNIT(X) MU_IF(MU_COUNT_ARG(X),const,)
-
 typedef struct METHODRETURN_DATA_TAG
 {
-    CONST_BY_COMPILATION_UNIT(METHODRETURN_C) int statusCode;
-    CONST_BY_COMPILATION_UNIT(METHODRETURN_C) char* jsonValue;
+    int statusCode;
+    char* jsonValue;
```

Comment:
I think this can stay `const`.

Reply:
We modify the value anyway.

D:\Repos\shelf1\azure-iot-sdk-c\serializer\src\methodreturn.c:72
                if (mallocAndStrcpy_s(&(result->data.jsonValue), jsonValue) != 0)

Reply:
And the compiler had a warning for that too:

   - WARN  | xcodebuild:  AzureIoTHubClient/serializer/src/methodreturn.c:74:39: warning: passing 'const char **' to parameter of type 'char **' discards qualifiers in nested pointer types [-Wincompatible-pointer-types-discards-qualifiers]